docs/framework/data/adonet/sql/linq/downloading-sample-databases.md

```diff
@@ -1,50 +1,48 @@
+
+1. Before you can use the Northwind database, you have to choose one of two options:
+
+    - Recreate the database by running the `instnwnd.sql` script file in the installation folder.
+
+    - Attach the `northwnd.mdf` file with its corresponding `*.ldf` log file.
+
+    Optionally, you can also install the Pubs sample database in the same way.
   
-2.  Go to the [Northwind and Pubs Sample Databases](https://go.microsoft.com/fwlink?linkid=64296) Web site.  
+## Download SQL Server Express  
+SQL Server Express is available without charge, and you can redistribute it with applications. If you are using Visual Studio, SQL Server Express is included in the Pro and higher editions.  
   
-3.  Click **Download**.  
+### To download and install SQL Server Express 
   
-4.  In the **File Download** dialog box, select **Save**.  
+1.  Go to the [SQL Server Express Editions](https://www.microsoft.com/sql-server/sql-server-editions-express) page.  
   
-5.  After the file has downloaded, double-click the **Nwind.exe** file to install the database.  
+1.  Select **Download now**. 
   
-     By default, the database is installed at *drive*:\SQL Server 2000 Sample Databases.  
+1.  After the file has downloaded, double-click the file and follow the installation instructions in the setup program.  
   
-## Downloading SQL Server Express Edition  
- SQL Server Express Edition is available without charge, and you can redistribute it with applications. If you are using Visual Studio, SQL Server Express Edition is included in the Pro and greater editions.  
+## Download SQL Server Management Studio
+If you want to modify a database that you have downloaded, you can access the database from **Server Explorer** in the Visual Studio integrated development environment (IDE), or you can use Microsoft SQL Server Management Studio (SSMS).  
   
-#### To download and install SQL Server Express Edition  
+### To download SQL Server Management Studio  
   
-1.  Start Internet Explorer.  
-  
-2.  Go to the [Microsoft Download Center](https://go.microsoft.com/fwlink?linkid=74602) Web site.  
-  
-3.  In the keywords box, type **SQL Server Express**.  
-  
-4.  Click **Go**.  
-  
-5.  On the results page, click the link to the **Microsoft SQL Server 2005 Express Edition** download page.  
-  
-6.  Follow the installation instructions on the Web site.  
-  
-## Downloading Management Studio Express  
- If you want to modify a database that you have downloaded, you can access the database from **Server Explorer** in the Visual Studio integrated development environment (IDE), or use Microsoft SQL Server Management Studio Express (SSMSE).  
-  
-#### To download Management Studio Express  
-  
--   Follow the instructions at the [SSMSE site](https://go.microsoft.com/fwlink/?LinkId=95933).  
+-   Follow the instructions at the [Download SQL Server Management Studio (SSMS)](https://docs.microsoft.com/sql/ssms/download-sql-server-management-studio-ssms) page.  
```

Comment:
link should be /sql/ssms/download-sql-server-management-studio-ssms, otherwise it doesn't work in the VS offline books

Reply:
I have just fixed this in a new commit.